lib/explorer/polars_backend/series.ex

```diff
@@ -38,12 +38,15 @@ defmodule Explorer.PolarsBackend.Series do
   def to_iovec(series), do: Shared.apply_series(series, :s_to_iovec)
 
   @impl true
+  def cast(%Series{dtype: :string} = series, {:datetime, precision}),
+    do: Shared.apply_series(series, :s_strptime, [nil, Atom.to_string(precision)])
+
```

Comment:
I wasn't sure if we wanted this logic here or in rust inside `s_cast`. I could see it going either way.

Reply:
Looks good to me here!

Reply:
If we follow that path of using the `ExTimeUnit`, we don't need to convert the atom to string here :)